### PR TITLE
Simplify the use of Docker containers to build bomsh/bomtrace/bomtrace2

### PR DESCRIPTION
**Pull request by edwarnicke**
Signed-off-by: Ed Warnicke <hagbard@gmail.com>

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -22,4 +22,17 @@ jobs:
...
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the code
+        uses: actions/checkout@v2
+      - name: docker build
+        run: docker run -it --rm -v ${PWD}:/out $(cd .devcontainer && docker build -q .)
+      - name: Check
+        run: |
+          ls bomsh
```
Comment (yonhan3) [inline .github/workflows/ci.yaml]: ls bombash

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,5 @@
 *.o
-src/hello
\ No newline at end of file
+src/hello
+bomsh
```
Comment (yonhan3) [inline .gitignore]: bombash

Comment (yonhan3) [reply .gitignore]: oh, actually, since the three files are put into ./out dir, so we can ignore the whole ./out dir?

Comment (edwarnicke) [reply .gitignore]: The out dir exists only in the container... not on the host.

**Review by yonhan3 — APPROVED** [review]

**Pull request merged**